atox/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,120 @@
+    <string name="message">Nachricht</string>
+    <string name="messages">Nachrichten</string>
+    <string name="messages_incoming">Eingehende Nachrichten</string>
+    <string name="name">Name</string>
+    <string name="name_default">aTox Benutzer</string>
+    <string name="no_contacts_call_to_action">Worauf wartest du? Füge einen Freund hinzu und starte sofort! </string>
+    <string name="navigation_drawer_close">Navigationsschublade schließen</string>
+    <string name="navigation_drawer_open">Navigationsschublade öffnen</string>
+    <string name="profile_photo_description">Profilbild</string>
+    <string name="reject">Ablehnen</string>
+    <string name="send">Senden</string>
+    <string name="sending">Wird gesendet…</string>
+    <string name="settings">Einstellungen</string>
+    <string name="status_indicator">Statusanzeige</string>
+    <string name="status_message">Status Nachricht</string>
+    <string name="tox_id">Tox-ID</string>
+    <string name="tox_id_error_length">Die Tox-ID ist immer 76 Zeichen lang (%1$d/76)</string>
+    <string name="tox_id_error_checksum">Prüfsumme ungültig</string>
+    <string name="tox_id_error_hex">Nur Hexadezimal erlaubt</string>
+    <string name="tox_id_share">Tox-ID teilen</string>
+    <string name="update">Aktualisieren</string>
+    <string name="quit">Beenden</string>
+    <string name="quit_confirm">Bist du sicher, dass du dich ausloggen willst?</string>
+    <!-- Friend request notification -->
+    <string name="friend_requests">Freundschaftsanfragen</string>
+    <string name="friend_request_from">Freundschaftsanfrage von %1$s</string>
+    <!-- Chat history clearing -->
+    <string name="clear_history">Verlauf löschen</string>
+    <string name="clear_history_cleared">Verlauf gelöscht</string>
+    <string name="clear_history_confirm">Bist du sicher, dass du den Verlauf mit %1$s löschen möchtest? </string>
+    <string name="delete_message_confirm">Bist du sicher, dass du die Nachricht löschen willst? \"%1$s\"?</string>
+    <string name="import_tox_save">Importieren</string>
+    <string name="import_tox_save_failed">Importieren hat nicht funktioniert: %1$s</string>
+    <string name="export_tox_save">Tox Profil exportieren</string>
+    <string name="tox_save_exported">Tox Profil exportiert</string>
+    <string name="never">Nie</string>
+    <string name="info">Information</string>
+    <string name="atox_offline">Offline</string>
+    <string name="atox_connected_with_tcp">Verbunden (TCP)</string>
+    <string name="atox_connected_with_udp">Verbunden (UDP)</string>
+    <string name="read_qr">QR</string>
+    <string name="pref_heading_theme">Design</string>
+    <string-array name="pref_theme_options">
+        <item>Wie System</item>
+        <item>Hell</item>
+        <item>Dunkel</item>
+    </string-array>
+    <string name="pref_heading_type">Typ</string>
+    <string-array name="pref_proxy_type_options">
+        <item>Keinen</item>
```

Comment:
I'm on the fence with this one. Somehow "Keinen" looks strange. Maybe "Keins". Or "Kein". I don't know.